.ci/docker/ubuntu/Dockerfile

```
@@ -40,7 +40,8 @@ ENV ANACONDA_PYTHON_VERSION=$ANACONDA_PYTHON_VERSION
 ENV PATH /opt/conda/envs/py_$ANACONDA_PYTHON_VERSION/bin:/opt/conda/bin:$PATH
 COPY requirements-ci.txt /opt/conda/requirements-ci.txt
 COPY ./common/install_conda.sh install_conda.sh
-RUN bash ./install_conda.sh && rm install_conda.sh
+COPY ./common/common_utils.sh common_utils.sh
+RUN bash ./install_conda.sh && rm install_conda.sh common_utils.sh
 RUN rm /opt/conda/requirements-ci.txt
```

Nit: this will result in less layers in the docker image. You can do the same for the other docker files too

```suggestion
RUN bash ./install_conda.sh && \
         rm install_conda.sh common_utils.sh && \
         rm /opt/conda/requirements-ci.txt
```